tests/phpunit/CRM/Utils/UrlTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Psr\Http\Message\UriInterface;
```

Comment:
This test looks exactly the same as the one you've added to the `CRM_Utils_System` class. Maybe here it would make more sense to test the `parseUrl()` and `unparseUrl()` method?

Reply:
I've moved it out of system - it covers parse & unparse implicitly